### ISSUE TITLE
/public/index.php not found or unable to stat

### ISSUE THREAD

**Issue by lakshmansrikanth** [sapplica/sentrifugo#37]
I have a Mac running latest El capitan! I have installed Apache, MySQL, PHP and I have other web apps working perfectly fine.

I successfully installed Sentrifugo, when I login I get a error in apache error log. When I try to do any sort form post I get 503 error. I have even tried reinstalling several time but the same error!

Error log from file - 
`[Wed Jun 29 17:03:26.166609 2016] [:error] [pid 2080] [client 127.0.0.1:53573] script '/Library/WebServer/Documents/projects/sentri/sentrifugo-master/public/index.php' not found or unable to stat, referer: http://projects/sentri/sentrifugo-master/index.php/employee/`

**Comment by SiNONiMiTY** [comment]
Check the `DocumentRoot` Directive in your Apache `httpd-conf` and `httpd-vhosts.conf`

Make sure they are pointing to the same path.

**Comment by lakshmansrikanth** [comment]
**httpd.conf**

> DocumentRoot
"/Library/WebServer/Documents"
<Directory "/Library/WebServer/Documents">
Options +Indexes +FollowSymLinks +MultiViews
MultiviewsMatch Any
AllowOverride All
Require all granted
</Directory>

**https-vhosts.conf**

> <VirtualHost*:80>
 ServerName projects
DocumentRoot "/Library/WebServer/Documents/projects"
</VirtualHost>

**Comment by zaili313** [comment]
I have the same issue: Application Error 503

Log:
[Tue Sep 27 11:31:49.828769 2016] [:error] [pid 9300:tid 1212] [client ::1:60442] script 'C:/UwAmp/www/sentri/public/index.php' not found or unable to stat, referer: http://localhost:88/sentri/index.php/timemanagement/projects/edit

Is there anyone could find the resolution?

Have check in the conf file:
DocumentRoot "C:/UwAmp/www/"
	ServerName "main-serveur"
	Alias "/mysql/" "C:/UwAmp/phpapps/phpmyadmin/"
	Alias "/mysql" "C:/UwAmp/phpapps/phpmyadmin/"
	Alias "/uwamp/" "C:/UwAmp/phpapps/uwamp/"
	Alias "/uwamp" "C:/UwAmp/phpapps/uwamp/"
	<Directory "C:/UwAmp/phpapps/phpmyadmin/">
		AllowOverride All
		Options FollowSymLinks Includes Indexes 
		Require local
	</Directory>
	<Directory "C:/UwAmp/phpapps/uwamp/">
		AllowOverride All
		Options FollowSymLinks Includes Indexes 
		Require local
	</Directory>
	<Directory "C:/UwAmp/www/">
		AllowOverride All
		Options FollowSymLinks Indexes 
		Require all granted
		
	</Directory>

**Comment by jfowlie** [comment]
I get the same problem with basically the same setup as above. I've tried symlinking index.php, checking modrewrite in .htaccess, but cannot get anything other than a blank page after moving off the first page of the configuration wizard.

**Comment by raghuiamsingh** [comment]
Can someone help with this issue?
Stuck on it for 4 days now.

**Comment by success4us** [comment]
> Can someone help with this issue?
> Stuck on it for 4 days now.

Were you able to find a solution? @raghuiamsingh

**Comment by raghuiamsingh** [comment]
@success4us Yes I was able to solve it, it turned out to be some missing PHP extensions. 

Check Packages
`php -m`

Match the required packages on the following link:
http://www.sentrifugo.com/installation-guide

Use the following commands/links to install the missing
```
sudo apt-get install php5.6-mysql php-gettext php5.6-mbstring php-mbstring php-xdebug libapache2-mod-php5.6
sudo apt-get install php5.6-bcmath
sudo apt-get install php5.6-xml 
sudo apt-get install php5.6-dev
sudo apt-get install php5.6-intl
https://www.digitalocean.com/community/questions/enable-php-5-5-opcache-on-ubuntu-14-04-with-nginx-and-php-fpm
sudo apt-get install php5.6-soap
Sudo apt-get install php5.6-zip
```
I would suggest running all the above commands.
